static/app/api.tsx

```diff
@@ -217,6 +217,10 @@ export type RequestOptions = RequestCallbacks & {
    * Values to attach to the body of the request.
    */
   data?: any;
+  /**
+   * Headers add to the request.
+   */
+  headers?: Record<string, any>;
```

Comment:
```suggestion
  headers?: Record<string, string>;
```

Reply:
caught a copypasta

Reply:
haha, as i was writing about it.... was able to draw from my own very recent examples